f-type/memory_maps.c

```diff
@@ -1,3 +1,5 @@
+#include <stddef.h>
```

Comment:
Should be in memory_maps.h

Reply:
No, because we don't use `NULL` in `memory_maps.h`. We only use it in `memory_maps.c` so it does not need the visibility of the .h